themes/bootstrap3/templates/RecordDriver/DefaultRecord/result-list.phtml

```diff
@@ -1,5 +1,5 @@
 <?php
-  $coverDetails = $this->record($this->driver)->getCoverDetails('result-list', 'medium', $this->recordLinker()->getUrl($this->driver));
+  $coverDetails = $this->record($this->driver)->getCoverDetails('result-list', 'medium', $this->recordLinker($this->results)->getUrl($this->driver));
```

Comment:
Not necessary, but just a thought: would this template be cleaner if we did a `$recordLinker = $this->recordLinker($this->results)` at the top and then abbreviated all the other calls to use the variable?

Reply:
Done while I added the parameter to other templates as well.